src/plugin/relativeTime/index.js

```diff
@@ -17,21 +47,23 @@ export default (o, c, d) => {
     y: 'a year',
     yy: '%d years'
   }
+  if (o.strict) {
+    Object.assign(d.en.relativeTime, {
```

Comment:
please avoid using es6 syntax `Object.assign`

Reply:
But `Object.keys` is used in [updateLocale](https://github.com/iamkun/dayjs/blob/dev/src/plugin/updateLocale/index.js#L6).

Reply:
`Object.keys` is not introduced in es6

Reply:
Ah, sorry about that, how about this `extend` util?